app/[category]/v/[prefix]/[videoId]/[lang]/page.tsx

```diff
@@ -55,22 +58,35 @@ export default async function Index({params}: {params: VideoParams}) {
     );
   }
 
-  const transcriptLangCode = params.lang === 'eng' ? 'en' : params.lang;
+  const transcriptLangCode = params.lang;
 
   // Handle the fact that transcript files for english still use the 2-letter ISO-639 code.
+  const loadData = new Array<Promise<any>>;
+  loadData.push(getMetadata(params.category, params.videoId));
+  loadData.push(await DiarizedTranscript.makeFromStorage(
```

Comment:
The `await` here looks like it shouldn't compile, what with loadData being an array of promises

Reply:
Apparently it does compile AND work... weird.  But fixed.